packages/dbml-parse/tests/interpreter/input/column_caller_type.in.dbml

```diff
@@ -0,0 +1,7 @@
+Table User {
+    id int
+    followers int(MAX)
+    subscribers int("MAX")
+    patrons int('MAX')
```

Comment:
Please consider using a valid column type when writing testcase, for example:
- [nvarchar(max)](https://learn.microsoft.com/en-us/sql/t-sql/data-types/nchar-and-nvarchar-transact-sql?view=sql-server-ver16#nvarchar---n--max--)
- [varchar(max)](https://learn.microsoft.com/en-us/sql/t-sql/data-types/char-and-varchar-transact-sql?view=sql-server-ver16#varchar---n--max--)
- [varbinary(max)](https://learn.microsoft.com/en-us/sql/t-sql/data-types/binary-and-varbinary-transact-sql?view=sql-server-ver16#varbinary---n--max--)

Reply:
@nguyenalter I've fixed this